viz_v2/wiki_categorize/src/bigQueryData.ts

```diff
@@ -0,0 +1,76 @@
+const BigQuery = require('@google-cloud/bigquery');
+
+export class bigQueryData {
+    private projectId: string;
+    private datasetID: string;
+    private originalDataTable: string;
+    private distDataTable: string;
+    
+    private bigquery: any;
+    private dataset: any;
+    private table: any;
+
+    constructor(config) {
+        this.projectId = config.projectId;
+        this.datasetID = config.datasetID;
+        this.originalDataTable = config.originalDataTable;
+        this.distDataTable = config.distDataTable;
+
+        this.bigquery = new BigQuery({
+            keyFilename: config.keyFilename,
+            projectId: this.projectId,
+        });
+
+        this.dataset = this.bigquery.dataset(this.datasetID);
+        this.table = this.dataset.table(this.distDataTable);
+    }
+
+    async querySourceTable() {
+
+        // max page ID 57726837
+        const sqlQuery =    `SELECT * 
+                                FROM \`${this.projectId}.${this.datasetID}.${this.originalDataTable}\`
+                            WHERE page_id >= 43295130`;
```

Comment:
where does this magic number come from? Add a comment to explain?

Reply:
Ah I was experimenting with this. This is the talk page categorization code separate from the vis, I'll clean it up.